utils/common/src/test/java/org/apache/brooklyn/util/text/StringEscapesTest.java

```diff
@@ -115,4 +117,80 @@ public void testJavaLists() {
             JavaStringEscapes.unwrapJsonishListIfPossible("\"\",,x,\"\""));
     }
 
+    @Test
+    public void testJavaListString() {
+        Assert.assertEquals(MutableList.of("hello", "world"),
+            JavaStringEscapes.unwrapQuotedJavaStringList("\"hello\", \"world\"", ","));
+        try {
+            JavaStringEscapes.unwrapQuotedJavaStringList("\"hello\", world", ",");
+            Assert.fail("Should have thrown");
+        } catch (Exception e) { /* expected */ }
+        
+        Assert.assertEquals(MutableList.of("hello", "world"),
```

Comment:
These assert calls are the wrong way round - first argument is `actual`, second argument is `expected` for testng.

Reply:
yeah ... i just copy-pasted other examples; sometimes when i encounter this i change it if it's irritating, but in this case i didn't (yet).